pkgs/development/python-modules/conda-libmamba-solver/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildPythonPackage,
+  pythonRelaxDepsHook,
+  fetchFromGitHub,
+  libmambapy,
+  hatchling,
+  hatch-vcs,
+  boltons,
+}:
+buildPythonPackage rec {
+  pname = "conda-libmamba-solver";
+  version = "24.1.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    inherit pname version;
+    owner = "conda";
+    repo = "conda-libmamba-solver";
+    rev = version;
+    hash = "sha256-vsUYrDVNMKHd3mlaAFYCP4uPQ9HxeKsose5O8InaMcE=";
+  };
+
+  nativeBuildInputs = [
+    pythonRelaxDepsHook
+    hatchling
```

Comment:
There's a new, cleaner, interface to `buildPythonPackage`: https://nixos.org/manual/nixpkgs/unstable/#handling-dependencies

`hatchling` would go in `build-system = [ ... ]`, and `propagatedBuildInputs` would be replaced with `dependencies`; this is not just about names, but also about supporting the future transition to a different way to handle python `dependencies`

Reply:
I've updated them.